### PR TITLE
feat: Add GraphQL tests

### PR DESCRIPTION
**Pull request by mayaisrani**
Updated Application.test.js to include test cases for new sub-mutation updates

Added new tests for fetching subsections of the larger application (i.e. FoodStampInfo, ProgramInfo)

### CODE REVIEW

File: oneapp-server/types/ApplicationContact.test.js [hunk 1]
```diff
@@ -1,11 +1,7 @@
-const { createTestClient, dataSources, services } = require('../__utils/TestingUtils');
-const client = createTestClient();
-const passwordGenerator = require('generate-password');
-
-jest.mock('generate-password');
+const { createTestClient, dataSources } = require('../__utils/TestingUtils');
```
Comment (mayaisrani) [inline oneapp-server/types/ApplicationContact.test.js:1]: removing unused variables

File: oneapp-server/types/ApplicationFoodStampInfo.js [hunk 2]
```diff
@@ -42,7 +42,7 @@ const typeDef = gql`
 
 const resolvers = {
   Application: {
-    foodStampInfo: (_parent, _args, { dataSources, auth, language }) => dataSources.FoodStampInfoDao.getFoodStampInfo(auth.user.USER_ID, language.code),
+    foodStampInfo: (_parent, _args, { dataSources, auth, language }) => dataSources.ApplicationFoodStampInfoDao.getFoodStampInfo(auth.user.USER_ID, language.index),
```
Comment (mayaisrani) [inline oneapp-server/types/ApplicationFoodStampInfo.js:45]: a couple bug fixes

File: oneapp-server/types/ApplicationProgramInfo.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ const typeDef = gql`
 
 const resolvers = {
   Application: {
-    programInfo: (_parent, _args, { dataSources, auth, language }) => dataSources.ApplicationProgramInfoDao.getProgramInfo(auth.user.USER_ID, language.code),
+    programInfo: (_parent, _args, { dataSources, auth, language }) => dataSources.ApplicationProgramInfoDao.getProgramInfo(auth.user.USER_ID, language.index),
```
Comment (mayaisrani) [inline oneapp-server/types/ApplicationProgramInfo.js:77]: another bug fix

**Review by ronaldheft-usds — APPROVED** [review]
Tests look great. Thanks! Going to merge so I can do an update on #463 .

**Pull request merged**